api/cmd/main.go

```diff
@@ -66,6 +68,7 @@ func main() {
 
 		backend.RegisterRouter(ws)
 		dataflow.RegisterRouter(ws)
+        block.RegisterRouter(ws)
```

Comment:
Why do we need to register at 2 places?

Reply:
It's required at one place only. Removed the other